package.json

```diff
@@ -28,11 +28,17 @@
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "devDependencies": {
+    "bookshelf": "^0.9.2",
+    "chai": "^3.0.0",
     "eslint": "^1.9.0",
     "eslint-config-lob": "^2.0.0",
     "generate-changelog": "^1.0.0",
-    "chai": "^3.0.0",
+    "hapi": "^11.1.2",
```

Comment:
FYI this plugin doesn't work with the latest version of Hapi because as of v12, Hapi no longer converts `?include[]=total_count` into `{ include: [ 'total_count' ] }` in `request.query` without using a plugin like [hapi-qs](https://github.com/daf-spr/hapi-qs). Hat tip to @charlesbjohnson for figuring that out.